File: examples/04_application/configuration/configuration.py
# Lean Code — BSD 3-Clause License — Vivian Voss, 2026
# Scope: Read application configuration.


def read_configuration_from_file(path):
    configuration = {}

    for line in open(path).read().strip().split("\n"):
        key, value = line.split("=", 1)
        configuration[key.strip()] = value.strip()

    return configuration

File: examples/04_application/configuration/configuration.js
// Lean Code — BSD 3-Clause License — Vivian Voss, 2026
// Scope: Read application configuration.

import { readFileSync } from "node:fs";

function readConfigurationFromFile(path) {
  const lines = readFileSync(path, "utf-8").trim().split("\n");
  const configuration = {};

  for (const line of lines) {
    const [key, value] = line.split("=", 2);
    configuration[key.trim()] = value.trim();
  }

  return configuration;
}

export { readConfigurationFromFile };
